bin/update-ips.bash

```diff
@@ -304,26 +327,37 @@ parse_url_port() {
   esac
 }
 
-# Updates the configuration to allow IPs.
+# Updates the configuration to allow CIDRs.
 #
-# Usage: allow_ips <config_file> <config_option> <ip> [<ip> ...]
-allow_ips() {
+# Usage: allow_cidrs <config_file> <config_options> <cidr> [<cidr> ...]
+allow_cidrs() {
   local config_file="${1}"
   local config_option="${2}"
   shift 2
 
-  local -a ips
-  readarray -t ips <<<"$(sort_ips "${@}")"
-
   local -a cidrs
-  readarray -t cidrs <<<"$(process_ips_to_cidrs "${config_file}" "${config_option}" "${ips[@]}")"
+  readarray -t cidrs <<<"$(sort_cidrs "${@}")"
 
   local list
   list=$(echo "[$(for v in "${cidrs[@]}"; do echo "${v},"; done)]" | yq4 -oj)
 
   yq_eval "${config_file}" "${config_option}" "${config_option} = ${list}"
 }
 
+# Updates the configuration to allow IPs.
+#
+# Usage: allow_ips <config_file> <config_option> <ip> [<ip> ...]
+allow_ips() {
+  local config_file="${1}"
+  local config_option="${2}"
+  shift 2
+
+  local -a cidrs
+  readarray -t cidrs <<<"$(process_ips_to_cidrs "${config_file}" "${config_option}" "${@}")"
+
+  allow_cidrs "${config_file}" "${config_option}" "${cidrs[@]}"
```

Comment:
Suggestion: Do we want to sort this before sending it? I can't see that the `process_ips_to_cidrs` does this, but I might be missing something.

```suggestion
  mapfile -t sortedCidrs < <(sort_cidrs "${cidrs[@]}")
  allow_cidrs "${config_file}" "${config_option}" "${sortedCidrs[@]}"
```

Reply:
It's sorted in `allow_cidrs`: https://github.com/elastisys/compliantkubernetes-apps/blob/e7e05d51347bc3cd407fc53542fbdc1cb5058bac/bin/update-ips.bash#L333-L339

Reply:
We also have this test that verifies that they are sorted: https://github.com/elastisys/compliantkubernetes-apps/blob/e7e05d51347bc3cd407fc53542fbdc1cb5058bac/tests/unit/general/bin-update-ips.bats#L115-L128

Reply:
Right, sorry. Yea I was looking at that function and noticed that it wasn't sorted here. But you're correct I didn't connect the dots there :sweat_smile: